docs/server-info.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[server-info]]
+== Server Information
+
+The APM Server exposes an API endpoint to query some server information.
+
+[[server-info-endpoint]]
+[float]
+=== Server Information endpoint
+Send an `HTTP GET` request to the server information endpoint:
+
+[source,bash]
+------------------------------------------------------------
+http(s)://{hostname}:{port}/
```

Comment:
for the other types we have a full curl example:

`curl http://localhost:8200/   --header "Content-Type: application/json`

I think having this plus the one bellow is a bit too verbose tbh

Reply:
I think the parity with the intake api docs is worth it, especially since this is such a short page anyway.